schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+  id INTEGER,
+  name VARCHAR(255),
+  date_of_birth DATE,
+  escape_attempts INTEGER,
+  neutered BIT,
```

Comment:
- [ ] Based on the project requirements, `neutered` should be of type `boolean -true or false` and not `BIT`. Kindly update this.